tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -36,6 +36,67 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("ISymWrapper"),
                                 WStr("Datadog.Trace"),
                                 WStr("MSBuild")};
+
+enum class DynamicInstrumentationErrorCode
```

Comment:
Do we really gain anything from having both an error code and potentially a string? It makes this file a bit hard to read and makes things a bit unclear. Should we maybe switch to only having a list of `const` strings here, and eliminate the error codes?

Reply:
I can eliminate the error codes, it's there to avoid specifying them manually and probably in the future have a piece of code that extracts the error code and lay out the reasoning in some kind of histogram.
Do you suggest me to place hard-coded and delete the enumeration?

Reply:
Yeah, I mean, there are constant strings and it's unlikely we will want to change the wording too often, so grabbing a histogram from telemetry based on the raw strings seems reasonable enough to me?

Reply:
> Do you suggest me to place hard-coded and delete the enumeration?

Reply:
Done in 8f3b029d9f707341ec353f2c187f0cfadd81a081.